From a6b2bebbd36324e0bc63c1a3a75162d9c80e8a15 Mon Sep 17 00:00:00 2001
From: Lucas Stach <l.stach@pengutronix.de>
Date: Thu, 29 Oct 2015 12:52:47 +0100
Subject: [PATCH 143/148] staging: etnaviv: remove etnaviv_gem_object paddr
 remnants

Those are not needed anymore since command submission uses the new
etanviv_cmdbuf now.

Signed-off-by: Lucas Stach <l.stach@pengutronix.de>
Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
---
 drivers/staging/etnaviv/etnaviv_drv.h | 1 -
 drivers/staging/etnaviv/etnaviv_gem.c | 9 ---------
 drivers/staging/etnaviv/etnaviv_gem.h | 7 -------
 3 files changed, 17 deletions(-)

diff --git a/drivers/staging/etnaviv/etnaviv_drv.h b/drivers/staging/etnaviv/etnaviv_drv.h
index 719e331..f838449 100644
--- a/drivers/staging/etnaviv/etnaviv_drv.h
+++ b/drivers/staging/etnaviv/etnaviv_drv.h
@@ -89,7 +89,6 @@ int etnaviv_gem_prime_pin(struct drm_gem_object *obj);
 void etnaviv_gem_prime_unpin(struct drm_gem_object *obj);
 void *etnaviv_gem_vaddr_locked(struct drm_gem_object *obj);
 void *etnaviv_gem_vaddr(struct drm_gem_object *obj);
-dma_addr_t etnaviv_gem_paddr_locked(struct drm_gem_object *obj);
 void etnaviv_gem_move_to_active(struct drm_gem_object *obj,
 		struct etnaviv_gpu *gpu, u32 access, u32 fence);
 void etnaviv_gem_move_to_inactive(struct drm_gem_object *obj);
diff --git a/drivers/staging/etnaviv/etnaviv_gem.c b/drivers/staging/etnaviv/etnaviv_gem.c
index 0e0c783..311fb6c 100644
--- a/drivers/staging/etnaviv/etnaviv_gem.c
+++ b/drivers/staging/etnaviv/etnaviv_gem.c
@@ -338,15 +338,6 @@ void *etnaviv_gem_vaddr(struct drm_gem_object *obj)
 	return ret;
 }
 
-dma_addr_t etnaviv_gem_paddr_locked(struct drm_gem_object *obj)
-{
-	struct etnaviv_gem_object *etnaviv_obj = to_etnaviv_bo(obj);
-
-	WARN_ON(!mutex_is_locked(&obj->dev->struct_mutex));
-
-	return etnaviv_obj->paddr;
-}
-
 void etnaviv_gem_move_to_active(struct drm_gem_object *obj,
 	struct etnaviv_gpu *gpu, u32 access, u32 fence)
 {
diff --git a/drivers/staging/etnaviv/etnaviv_gem.h b/drivers/staging/etnaviv/etnaviv_gem.h
index f4330da..7a99fd1 100644
--- a/drivers/staging/etnaviv/etnaviv_gem.h
+++ b/drivers/staging/etnaviv/etnaviv_gem.h
@@ -69,19 +69,12 @@ struct etnaviv_gem_object {
 	struct sg_table *sgt;
 	void *vaddr;
 
-	/* for ETNA_BO_CMDSTREAM */
-	dma_addr_t paddr;
-
 	/* normally (resv == &_resv) except for imported bo's */
 	struct reservation_object *resv;
 	struct reservation_object _resv;
 
 	struct list_head vram_list;
 
-	/* for buffer manipulation during submit */
-	bool is_ring_buffer;
-	u32 offset;
-
 	/* cache maintenance */
 	u32 last_cpu_prep_op;
 
-- 
2.6.1

